salt/tests/README.md

```diff
@@ -0,0 +1,23 @@
+# Salt Tests
+
+## Unit
+Unit tests are used to test each Salt custom functions in MetalK8s.
+
+It uses [unittest](https://docs.python.org/2/library/unittest.html)
+and [SaltTesting](https://github.com/saltstack/salt-testing) that
+provide some helpers to write tests for Salt modules, pillars, ...
+
+
+### Run tests using tox
+
+```
+tox -e unit-tests
```

Comment:
No room for markers here I think...
So each time we initiate this, we must run the entire test suite?

Reply:
I just need to change the tox.ini and you can do classic pytest filtering like `tox -e unit-tests -k "not kubernetes"` (that filter out every test with `kubernetes` in the name)